serverless.yml

```diff
@@ -0,0 +1,235 @@
+# Welcome to Serverless!
+#
+# This file is the main config file for your service.
+# It's very minimal at this point and uses default values.
+# You can always add more config options for more control.
+# We've included some commented out config examples here.
+# Just uncomment any of them to get that config option.
+#
+# For full config options, check the docs:
+#    docs.serverless.com
+#
+# Happy Coding!
+
+service: rs-store-app
+# app and org for use with dashboard.serverless.com
+#app: your-app-name
+#org: your-org-name
+
+# You can pin your service to only deploy with a specific Serverless version
+# Check out our docs for more details
+frameworkVersion: '2'
+
+provider:
+  name: aws
+  runtime: nodejs12.x
+#  profile: maksimKirilkoTest
+
+plugins:
+  - serverless-finch
+  - serverless-single-page-app-plugin
+
+custom:
+  client:
+    bucketName: kirilko-app2
+    distributionFolder: build
+    indexDocument: index.html
+    manageResources: false
```

Comment:
If you set `manageResources` to `false` here, then there is no need to add `--no-config-change --no-policy-change --no-cors-change` to the command line (serverless-finch "manages" your resources by default, this is what you are opting out of by adding those switches)

Reply:
Thanks